CMakeLists.txt

```diff
@@ -114,10 +140,10 @@ if(NOT DEFINED LevelZero_LIBRARY)
   endif()
 endif()
 
-message(STATUS "OpenCL_LIBRARY: ${OpenCL_LIBRARY}")
+message(STATUS "OpenCL_FOUND: ${OpenCL_FOUND}")
```

Comment:
This is obviously ugly; I'll have to figure out how to show the path with import targets.
Maybe it's also enough to report YES / NO for each of these.